Sources/VLCActionSheet/VLCActionSheetCell.swift

```diff
@@ -47,7 +50,7 @@ class VLCActionSheetCell: UICollectionViewCell {
         setupViews()
     }
 
-    private func setupViews() {
+    func setupViews() {
```

Comment:
I don't think this method should be internal.

Reply:
can I ask you why??

Reply:
I believe that it would be quite wrong to let other classes have the possibility to initialize the properties of `VLCActionSheetCell`.

I'm not saying that `VLCActionSheetCell` is perfect but I think classes should try to stay "self-contained".

For example, I think this would be nicer:

```
class TestCell: VLCActionSheetCell {
    override init(frame: CGRect) {
        super.init(frame: frame) // super.setupViews() is called in the super.init()
        setupViews()
    }

    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }

    private func setupViews() {
        stackView.removeArrangedSubview(icon)
        stackView.addArrangedSubview(//something\\)
    }
}
```

Also, I'm sorry if I don't make any sense 😅

Reply:
Gotcha! And it totally makes sense :D